packages/material-ui-styles/src/makeStyles/makeStyles.js

```diff
@@ -242,7 +242,7 @@ export default function makeStyles(stylesOrCreator, options = {}) {
       React.useDebugValue(classes);
     }
     if (process.env.NODE_ENV !== 'production') {
-      const whitelistedComponents = ['MuiButton'];
+      const whitelistedComponents = ['MuiButton', 'MuiTypography'];
```

Comment:
```suggestion
      const allowlistedComponents = ['MuiButton', 'MuiTypography'];
```

Reply:
Let's see if we can fix the component that does not follow this pattern first and get rid of this list

Reply:
@mnajdova Agreed, let's not even go there. I'd rather we didn't get bogged down in a debate of what term does or doesn't have racist etymology on the one hand, or be accused of slacktivism on the other with token changes such as this. (If you'll forgive the pun.)

Reply:
Ok, I did quick round, there are many consistencies that I believe it would be easier to be resolved by per component PRs anyway. Let's keep this list until necessary and in the end we can drop it...